frontend/src/components/pages/CreateReview/CreateReview.tsx

```diff
@@ -152,9 +170,9 @@ const CreateReview = ({ id }: CreateReviewProps): React.ReactElement => {
   /**
    * Function to be called when the review is published.
    */
-  const onPublish = async () => {
+  const onPublish = async (save = false) => {
```

Comment:
Could you change the name/doc of this function to reflect the dual functionality?

Reply:
Still was not completely sure how to name it, since the function is also used for both inserting/editing reviews so let me know if the naming could be improved

Reply:
Perhaps something like `onSubmit`? Either that or maybe `onSave` with the parameter being `publish`